File: 4. Java/EX35.java
package Apostila;

import java.util.Locale;
import java.util.Scanner;

public class EX35
{
    public static void main(String[] args)
    {
        // Variáveis
        int n, qtd, den = 0, fora = 0; 

        // Lê a quantidade de números
        try (Scanner ent = new Scanner(System.in))
        {
            // Lê a quantidade de números
            System.out.print("# Quantos numeros voce vai digitar: ");
            qtd = ent.nextInt();
            
            // Laço para ler números
            for (int i = 0; i < qtd; i++) 
            {
                // Lê um número
                System.out.print("# Digite um numero: ");
                n = ent.nextInt();
                
                // Verifica intervalo
                if ((n >= 10) && (n <= 20))
                {
                    den++; // Dentro
                }
                else
                {
                    fora++; // Fora
                }
            }
        }
        
        // Resultados
        System.out.println("# " + den + " DENTRO");
        System.out.println("# " + fora + " FORA");
    }
}

File: 6. Python/EX35.py
# Inicializa contadores
fora = 0
dentro = 0

# Solicita a quantidade de números a serem digitados
qtd = int(input("# Quantos numeros voce vai digitar: "))

# Loop para leitura dos números
for i in range(qtd):
    n = int(input("# Digite um numero: "))

    # Verifica se o número está entre 10 e 20
    if 20 >= n >= 10:
        dentro += 1  # Conta como dentro
    else:
        fora += 1    # Conta como fora

# Exibe resultados
print(f"# {dentro} DENTRO")
print(f"# {fora} FORA")
